make_level.py

```diff
@@ -141,38 +116,49 @@ def main():
         "level_name: {}".format(level_name),
         "Confirm[y/n] "
     ]), end='')
-
-    choice = input().lower()
-    if choice == 'n':
-        return
+    
+    choice = ''
+    while choice != 'y':
+        choice = input().lower()
+        if choice == 'n':
+            return
+        elif choice != 'y':
+            print("Confirm[y/n] ", end='')
 
     # Register package
     # Read setup.py
     with open("setup.py", 'r') as fp:
         filedata = fp.read()
 
-    # Update desired lines
+    # Check if skill is already registered.
+    skill_exists = False
+    if "gitgud.skills.{}".format(skill_name) in filedata:
+        skill_exists = True
+
+    # Concatenate potential entry
     replace1 = "\n".join([
```

Comment:
Indent both replaces, we only use them if the if statement executes

Reply:
Changed